scripts/execute-vscode-sideload.ps1

```diff
@@ -0,0 +1,38 @@
+param (
+    [string]
+    [Parameter(Mandatory = $false)]
+    $Platform = "win-x64",
+    [switch]
+    $WSL
+)
+
+
+Push-Location ./vscode/microsoft-kiota
+
+# Install all dependencies
+npm i
+
+# Package the VS Code extension
+vsce package
```

Comment:
you could add the installation of this or use `npx @microsoft/vsce package` instead

Reply:
I actually love this idea. I'll update.